pkg/fqdn/dnsproxy/proxy_test.go

```diff
@@ -1028,39 +1029,79 @@ func (t selectorMock) String() string {
 }
 
 func Benchmark_perEPAllow_setPortRulesForID(b *testing.B) {
+	// This benchmark loops through nEPs number of endpoints ~to distinct pods;
+	// - With nMatchPatterns number of matchPatterns in their DNS config
+	// - With nMatchName number of matchNames in their DNS rules
+	// - Where every everyNIsEqual EPs have the same DNS rules
+	// - Where every everyNHasWildcard EPs have a wildcard inserted
 	const (
-		nMatchPatterns = 100
```

Comment:
I chose all these values kinda arbitrarily, leaning towards the values we are seeing in some clusters. It's a bit overkill with 600+ matchNames in the allowlist per pod, but it is what it is. It also makes it a lot easier to actually track down the allocations.

Reply:
Could you put your justification as a code comment or in the commit msg?

Reply:
Yeah, can do 👌